src/test/java/com/egov/business-services/preTests/billingServiceDemandPretest.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Business services - billing service demand calls
+
+Background:
+  * def jsUtils = read('classpath:jsUtils.js')
+  * def billingServiceDemandConstants = read('../constants/billing-service-demand.yaml')
+  * def commonConstants = read('../constants/commonConstants.yaml')
+  * def createDemandRequest = read('../requestPayload/billing-service-demand/create.json')
+  # commenting as this will be added later
+  # * def updateDemandRequest = read('../requestPayload/billing-service-demand/update.json')
+  # * def searchDemandRequest = read('../requestPayload/billing-service-demand/search.json')
+  * configure headers = read('classpath:websCommonHeaders.js')
+
+@successCreate
+Scenario: Create Demand success Call
+  * print createDemandRequest
+  Given url createDemandUrl
+  And request createDemandRequest
+  When method post
+  Then status 201
+  And def billingServiceDemandResponseHeader = responseHeaders
+  And def billingServiceDemandResponseBody = response
+
+@errorCreate
+Scenario: Create Demand error Call
+  * print createDemandRequest
+  Given url createDemandUrl
+  And request createDemandRequest
+  When method post
+  Then assert responseStatus == 400 || responseStatus == 403
```

Comment:
Is the `responseStatus` not static? means either only `400` or only `403`?

Reply:
it varies based on request payload. this condition is used because i don't need to write a separate scenario for each status code

Reply:
Got you, then I think in that case you can validate the status code from your individual test itself, because if in future some discrepancy occurred on the status code this pretest will pass that test. This is what I fill.